io-engine/src/subsys/config/mod.rs

```diff
@@ -227,6 +227,6 @@ impl Config {
         assert!(self.socket_opts.set());
         assert!(self.iobuf_opts.set());
 
-        debug!("{:#?}", self);
+        info!("{:#?}", self);
```

Comment:
Does this need to be info?

Reply:
Yes, otherwise debugging SB we don't see the full config by default